src/interfaces/token.ts

```diff
@@ -113,6 +113,35 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
       try {
         const zapperBalances = await this.yearn.services.zapper.balances(account);
         balances.zapper = zapperBalances.filter(({ address }) => addresses.zapper.has(address));
+        const balance = await this.ctx.provider.read.getBalance(account);
+        const priceUsdc = await this.yearn.services.oracle.getPriceUsdc(EthAddress);
```

Comment:
Just verify this call works properly, since I remember we tried this with fantom network native token and it didnt. Some contracts have a specific functions for native tokens.

Reply:
I tried, for ETH it works correctly :)

Reply:
cool then :)